### PR TITLE
Updated homepage topics

### PR DESCRIPTION
**Pull request by dwnoble**
Screenshots:

Screenshot 2023-07-31 at 3 36 28 PM

### CODE REVIEW

File: static/js/apps/explore_landing/topics.json [hunk 1]
```diff
@@ -290,8 +317,8 @@
         ]
       }
     },
-    "education": {
-      "title": "Education",
+    "schools-housing-commute": {
+      "title": "Schools, housing, and commute",
```
Comment (pradh) [inline static/js/apps/explore_landing/topics.json:321]: nit: do we capitalize Housing and Commute?

File: static/js/apps/explore_landing/topics.json [hunk 2]
```diff
@@ -1,15 +1,42 @@
...
+      "examples": {
+        "general": [
+          {
+            "title": "Economic equity in California",
+            "url": "oq=Economic+equity+in+California"
+          },
+          {
+            "title": "Incarceration rates by race in the US",
+            "url": "oq=Incarceration+rates+by+race+in+the+US"
+          },
```
Comment (pradh) [inline static/js/apps/explore_landing/topics.json:23]: We have topics for these so we can append stuff like:

&t=dc/topic/EconomicEquity&p=geoId/06
&t=dc/topic/JusticeSystemEquity&p=country/USA
&t=dc/topic/HealthEquity&p=geoId/06085

**Review by pradh — APPROVED** [review]
Thanks Dan!

File: static/js/apps/explore_landing/topics.json (hunk 1 above)
Comment (dwnoble) [reply static/js/apps/explore_landing/topics.json:321]: that sounds good to me. added.

File: static/js/apps/explore_landing/topics.json (hunk 2 above)
Comment (dwnoble) [reply static/js/apps/explore_landing/topics.json:23]: added

**Pull request merged**